### DISCUSSION TITLE
Help with Fibula Cut Box Placement

### DISCUSSION THREAD

**Discussion by qferreira-1** [lassoan/SlicerBoneReconstructionPlanner#102] [General]
Hello All,

I have a simple case but for some reason the fibula cut box is continuously getting too close to the fibula bone. This would be biggerMitreBox0_B, but I’ve followed similar steps for previous cases and have not had this issue. I created the fib line and centred it, and re did this step to ensure it’s correctly centred. Any help would be amazing.
![724FC24E-8317-4768-89E8-2A1461268E6D](https://user-images.githubusercontent.com/122468415/232071195-03b2048d-5051-44b7-8448-3b538cae59da.jpeg)
![image](https://user-images.githubusercontent.com/122468415/232071377-e24668b1-3120-4bee-8e03-22098eb0fddc.jpg)

Thanks for your help in advance.

**Comment by qferreira-1** [comment]
Using Slicer 5.03 but haven’t had this issue previously.

**Comment by mauigna06** [comment]
Hi @qferreira-1 

> I have a simple case but for some reason the fibula cut box is continuously getting too close to the fibula bone. 

That's probably because you forgot to wrap-solidify the fibula segment. 
See 5. [here](https://github.com/SlicerIGT/SlicerBoneReconstructionPlanner/#segmentation-preparation-for-virtual-surgical-planning)

**Reply by qferreira-1** [reply]
Thank you for pointing that out, will try now.
